TestApp/app/src/main/java/com/example/testapp/viewmodel/LoginViewModel.kt

```diff
@@ -0,0 +1,4 @@
+package com.example.testapp.viewmodel
+
```

Comment:
this login view model should be a part of the login package

Reply:
updated under ui.login package